build

```diff
@@ -66,6 +66,32 @@ replace(path.join(__dirname, '.server-dist'), [
 	{regex: new RegExp(`(require\\\(.*?)(lib)(.*?\\\))`, 'g'), replace: `$1.lib-dist$3`},
 ]);
 
+var databaseExisted = true;
+
+try {
+	require.resolve('./database/sqlite.db');
+} catch (err) {
+	if (err.code !== 'MODULE_NOT_FOUND') throw err; // should never happen
+	databaseExisted = false;
+	console.log('Database does not exist. Creating one...');
+	fs.writeFileSync(path.resolve(__dirname, 'database/sqlite.db'), '');
+}
+
+var sqlite = require('sqlite');
+
+sqlite.open('./database/sqlite.db').then(function (database) {
+	if (!databaseExisted) {
+		var schemas = fs.readdirSync('./database/schema/');
+		for (var i = 0; i < schemas.length; i++) {
+			var sql = fs.readFileSync('./database/schema/' + schemas[i]).toString();
+			database.exec(sql);
+		}
+	}
+	if (fs.readdirSync('./database/migrations/').length) {
```

Comment:
it seems like this expects `-- Up` and `-- Down` comments?

Reply:
Yeah, it does. But the parser is built in.

Reply:
I should probably add a `DATABASE.md` to document the design.

Reply:
My point is that your migrations files does not have one. I noticed thats because you consider it a 'schema', but you must migrate all your schemas, so database/schema should not exist...

Reply:
right.